src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -721,6 +721,19 @@ StatefulLLMPipeline::StatefulLLMPipeline(
         ModelConfigDesc model_desc = get_modeldesc_from_json(models_path / "config.json");
         ov::AnyMap properties = config;
         auto compiled = setupAndCompileModel(model, model_desc, properties);
+        // Also export compiled model if required
+        const auto export_blob = pop_or_default(properties, "EXPORT_BLOB", false);
```

Comment:
What kind of path can be provided there?

1) If it's absolute path e.g `C:\workspace\openvino_model.blob` - then, I believe we shouldn't combine it with `model_path`.
2) If it's relative path, I believe the handling is right.

BTW, should the user provide path to the file or folder? How do you check this?

Reply:
Why not `DUMP_BLOB`?

Reply:
Only full path is accepted.
I only combine `models_path / "openvino_model.blob"` if the provided `blob_path` is empty.
Added a check that `.blob` file is provided

Reply:
> Only full path is accepted.

Perhaps it can be checked: https://en.cppreference.com/w/cpp/filesystem/path/is_absrel

Reply:
I'd like to check it before I try to create a file

Reply:
Makes sense